firmware/console/binary_log/log_field.cpp

```diff
@@ -34,7 +34,7 @@ void LogField::writeHeader(Writer& outBuffer) const {
 	buffer[45] = 0;
 
 	// Offset 46, length 4 = Scale
-	copyFloat(buffer + 46, 1.0f / m_multiplier);
```

Comment:
hold on why do we change this? is this because what we've called mult was actually a divider?

Reply:
I'm just reusing the word.  If you note in the constructor I'm now assigning `divisor/multiplier` to m_multiplier.  I suppose I could have changed the name to m_divisor?